ghost/portal/src/components/Frame.styles.js

```diff
@@ -385,9 +385,9 @@ const FrameStyles = `
     align-items: center;
     line-height: 0;
     border-radius: 4px;
-    background: #ffffff;
+    background: var(--white);
```

Comment:
This logo is marketing material which should be the same everywhere. These values should remain hardcoded.